docs: change parameter community_tax and min_deposit (#1)
medibloc/panacea-governance: proposals/2021-10-community_pool-minimum_deposit/README.md

### Abstract

We would like to change two parameters as follows.
- Change the community tax from 0% to 2%.
- Change the minimum proposal deposit from 100,000 MED to 10,000 MED.

### Background

Community pool is a means of funding participants who contribute to the Panacea ecosystem.<br/>

> As other Cosmos-based chains, we would like to use the community pool through governance votings to support contributions which boost the Panacea ecosystem.<br/>

Reviewer: ```suggestion
As other Cosmos-based chains, we would like to use the community pool through governance votings to support contributions which boost the Panacea ecosystem, such as outstanding improvements, events and marketing.<br/>
```
What do you think about adding brief examples as above, even though we already posted detail examples in the github issue?

Author: Oh, Good!
I think it's better to understand.

Author: I had a vague idea that it would be good to put the part you mentioned somewhere.

Reviewer: Yeah. For me, either ways is fine if it makes people understand clearly.

You can find various use-cases of the community pool from other Cosmos-based chains: https://github.com/medibloc/panacea-core/issues/186#issuecomment-927537658. 

> Although staking rewards of validators and delegators will be slightly reduced since the 2% of transactions fees will go to the community pool, the raised funds in the community pool will surely accelerate the Panacea ecosystem.

Reviewer: ```suggestion
Staking rewards of validators and delegators will be slightly reduced since the 2% of staking rewards (= block provision + tx fees) will go to the community pool. But, the raised funds in the community pool will surely accelerate the Panacea ecosystem.
```
The community tax affects the reward distribution by the `AllocateTokens()`:
https://github.com/cosmos/cosmos-sdk/blob/33e7297c791980428d0c698989ffa760284258f2/x/distribution/keeper/allocation.go#L81

It was confusing that the community tax ratio is applied to the `feeCollected` value. But, I also found that the block provision is also added to the `feeCollected`: 
https://github.com/cosmos/cosmos-sdk/blob/dda9bd8333774d70d330f43fa7f599e85f6c26aa/x/mint/abci.go#L37

In addition, we suggest lowering the minimum proposition to induce active participations in governance proposals.<br/>
Currently, the minimum cost to submit a proposal is 100,000 MED which is quite big.<br/>

> So, we propose to change it to 10,000 MED to lower a hurdle to register proposals.

Reviewer: ```suggestion
So, we propose to change it to 10,000 MED to lower a hurdle to register proposals (600 USD approximately on Sep 2021).
```
Accidentally, I erased the approximate amount in fiat. Pls commit this if you think it's necessary.

### Expected effects

> - The 2% of transaction fees will go to the community pool which can be spent only through governances.

Reviewer: ```suggestion
- The 2% of staking rewards will go to the community pool which can be spent only through governances.
```
Similar as https://github.com/medibloc/panacea-governance/pull/1/files#r717261657

> - The 2% of transaction fees will go to the community pool which can be spent only through governances.
>     - Staking rewards of validators and delegators will be slightly reduced.

Reviewer: ```suggestion
- The 2% of staking rewards will go to the community pool which can be spent only through governances.
```
Similar as https://github.com/medibloc/panacea-governance/pull/1/files#r717261657
